gdcdictionary/schemas/clinical_trial.yaml

```diff
@@ -0,0 +1,330 @@
+$schema: "http://json-schema.org/draft-04/schema#"
+
+id: "clinical_trial"
+title: Clinical Trial
+type: object
+namespace: https://niaid-clinical-trials-dictionary.org
+category: data_file
+program: '*'
+project: '*'
+description: >
+  The collection of all data related to a clinical trial.
+additionalProperties: false
+submittable: true
+validators: null
+
+systemProperties:
+  - id
+  - project_id
+  - created_datetime
+  - updated_datetime
+  - state
+
+links:
+  - exclusive: false
+    required: true
+    subgroup:
+    - name: core_metadata_collections
+      backref: data_files
```

Comment:
i think the backref should be `clinical_trials`?

Reply:
Yes, corrected it.